maintainers/depmgmt.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: default
+nav_order: 4
+parent: RAPIDS Maintainer Docs
+title: Dependency Management
+---
+
+# Dependency Management Documentation
+
+## Overview
+
+Outlines how to test and change dependencies of the RAPIDS projects
+
+### Intended audience
+
+Developers
+{: .label .label-green}
+
+Project Leads
+{: .label .label-blue}
+
+Operations
+{: .label .label-purple}
+
+## Adding a Dependency
+
+The [RAPIDS Integration repository](https://github.com/rapidsai/integration) contains all the information needed for adding a dependency to the RAPIDS projects. The `meta.yaml` you change should be based on the purpose of the package you are installing. The purpose of each meta-package is listed below:
+
+Package Name | Purpose
+--- | ---
+`rapids-build-env` | Installs all `conda` build dependencies to build & test RAPIDS libraries from source
+`rapids-doc-env` | Installs all tools needed to build RAPIDS documentation
+`rapids-notebook-env` | Installs a Jupyter Notebook server and other dependencies to run RAPIDS example notebooks; used in the `runtime` [stable](https://hub.docker.com/r/rapidsai/rapidsai/tags?page=1&name=runtime) and [nightly](https://hub.docker.com/r/rapidsai/rapidsai-nightly/tags?page=1&name=runtime) RAPIDS containers.
+
+### Adding Versions
+
+For new packages or those that do not have defined versions they need to be
+added.
+
+#### Modifying Recipes
+
+To add a package with versioning to the recipe we need the `PACKAGE_NAME` and
```

Comment:
similar comment as above. we should probably avoid first person pronouns like `we`. maybe change this to something like:

_To add a package with versioning to the recipe, the PACKAGE_NAME and the VERSIONING_NAME need to be added to the file._

there are a few other `we`'s and `you`'s in the doc that should also be addressed.